libafl_bolts/src/llmp.rs

```diff
@@ -2494,6 +2495,21 @@ where
             let pageinfo = (*msg).buf.as_mut_ptr() as *mut LlmpPayloadSharedMapInfo;
             (*pageinfo).shm_str = *shmem_description.id.as_array();
             (*pageinfo).map_size = shmem_description.size;
+            (*pageinfo).restarter_id = restarter_id;
```

Comment:
This will change the alignment of the page, I _think_ it was aligned to something. Be careful(?)

Reply:
if i understands correctly, it should be fine as long as we have #[repr(C)] right?

Reply:
it was 28 bytes before.
i don't think it was aligned to anything

Reply:
it's 32bytes on x64 28 bytes on x86 now,

Reply:
Yes should be fine I got confused